### ISSUE TITLE
$!eigenstates may contain a BOOTArray

### ISSUE THREAD

**Issue by gfldex** [lizmat/eigenstates#3]
```
use eigenstates;

my @dates = Date.new(2022, any(1..12), *).&eigenstates;
```
fails with:

```
No such method 'List' for invocant of type 'BOOTArray'.  Found 'List'
on type 'Any'
  in sub eigenstates at /usr/local/src/rakudo/install/share/perl6/site/sources/EDAA4AE0C8813633A2EA392374FB72F6B4D61047 (eigenstates) line 4
  in block <unit> at tmp/eigenstates-bugs.raku line 3
```

**Comment by gfldex** [comment]
The following works:
```
sub eigenstates(Junction:D \j){ 
    use nqp; 
    Array.new: nqp::getattr(j, Junction, '$!eigenstates')
};
```
I'm not sure if `Array` is the right thing to return tho.

**Comment by lizmat** [comment]
Fixed with version 0.10.   Thanks for the report!
